src/index.js

```diff
@@ -103,8 +103,8 @@ function allFederalHolidaysForYear(
       const dow = holiday.date.getUTCDay();
 
       if (dow == 0 && shiftSundayHolidays) {
-        // Actual holiday falls on Sunday.  Shift
-        // the observed date forward to Monday.
+        // Actual holiday falls on Sunday.
+        // Shift the observed date forward to Monday.
```

Comment:
These are fine, but FWIW, they may get changed again later.  The 18F style preference is to wrap at around the 80th column, so both the original and the modified are very minor violations.  This is what I get for not setting up a code formatter on this project.  😬

Reply:
Would you be open to use ESlint? I can open a PR and set it up for you :)

Reply:
Also, I think I might not be understanding the wrapping-at-80 rule very well. I counted how many characters there are on both lines and it looks like it's 42 and 53. Am I missing something? Are indentations counted differently?

Reply:
No, you're understanding correctly.  It wasn't right in the original code.  :)  I'll send up a PR in a little bit that includes eslint, prettier, and CI/CD

Reply:
Great! I'm a sucker for ESLint :)